api/src/org/apache/cloudstack/api/command/user/vpc/RestartVPCCmd.java

```diff
@@ -66,14 +66,14 @@ public Boolean getCleanup() {
         if (cleanup != null) {
             return cleanup;
         }
-        return true;
+        return false;
     }
 
     public Boolean getMakeredundant() {
         if (makeredundant != null) {
```

Comment:
Is there is a default value, why not use a primitive value with a default of `false`? Then, you do not need these logics in a POJO get method.

Reply:
@rafaelweingartner when the option is not passed, the value should be false which is what the fix it.